cdap-stream-clients/python/cdap_stream_client/streamclient.py

```diff
@@ -23,27 +23,22 @@
 
 class StreamClient(ConnectionErrorChecker):
 
-    __GATEWAY_VERSION = u'/v2'
     __REQUEST_PLACEHOLDERS = {
         u'streamid': u'<streamid>'
     }
-    __REQUESTS = {u'streams': __GATEWAY_VERSION + u'/streams'}
+    __REQUESTS = {u'streams': u'/streams'}
```

Comment:
This is because in v3/ there is a namespace between the version and the stream, so we cannot construct the URL in the same way. The service connector now connects to a v3 namespace, which is part of the connection config.